src/core/init.js

```diff
@@ -73,12 +73,9 @@ var rootjQuery,
 
 					if ( elem ) {
 						// Inject the element directly into the jQuery object
-						this.length = 1;
 						this[0] = elem;
+						this.length = 1;
```

Comment:
Or just for the consistency - https://github.com/jquery/jquery/pull/2000/files#diff-85f162ae43172ebe26b51a4d1c5d4eddR94